extensions/apiconnect/src/types/oas-extensions/x-ibm-configuration/assembly/execute/websocket-upgrade.policy.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  DataPowerAPIGateway as ExecuteDataPowerAPIGateway,
+  V200 as ExecuteV200,
+  V200Min,
+} from '.';
+
+export type WebSocketUpgradePolicy = DataPowerAPIGateway;
```

Comment:
~~Couldn't find in the documentation what this is a child of.~~

See other review comment.

Reply:
Fixed in https://github.com/loopbackio/loopback-next/pull/8105#discussion_r841824693